starky/src/cross_table_lookup.rs

```diff
@@ -1041,26 +971,21 @@ pub mod debug_utils {
         }
     }
 
-    fn check_ctl<F: Field>(
+    fn check_ctl<F: PrimeField64>(
```

Comment:
I'm not sure these functions are actually in use anywhere?  They don't seem to have been updated to reflect the greater capabilities of logup vs the old product argument.